Formula/coreutils.rb

```diff
@@ -28,6 +29,8 @@ class Coreutils < Formula
     depends_on "xz" => :build
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
```

Comment:
Add a comment that these two dependencies are for the patch, and that they should be removed when the patch is no longer needed.

Reply:
6aa86dd7b8b